src/repositories/qfRoundRepository.ts

```diff
@@ -11,6 +13,84 @@ export const findAllQfRounds = async (): Promise<QfRound[]> => {
     .getMany();
 };
 
+export enum QfArchivedRoundsSortType {
+  allocatedFund = 'allocatedFund',
+  totalDonations = 'totalDonations',
+  uniqueDonors = 'uniqueDonors',
+  beginDate = 'beginDate',
+}
+
+registerEnumType(QfArchivedRoundsSortType, {
+  name: 'QfArchivedRoundsSortType',
+  description: 'The attributes by which archived rounds can be sorted.',
+});
+
+@ObjectType()
```

Comment:
I'd Add some indexes to things like Name, slug, to facilitate adminjs searchs.

The Id column must be of type   @PrimaryGeneratedColumn()
which also sets and index.